src/client/components/ModalNew.tsx

```diff
@@ -171,7 +170,7 @@ const Footer = styled.div<ModalFooterProps>`
   align-items: center;
   justify-content: center;
   ${({ sticky }) =>
-    !sticky &&
+    sticky &&
```

Comment:
haha just wondering why this changes?

Reply:
It's just that before `position: sticky` (and "associate" styles) were being applied when `sticky` prop was `false` which is misleading is my opinion: you want to add the styles when "you ask" for them - `sticky` being `true`.

It was working before because `sticky` prop wasn't being provided to <Footer /> component: `!undefined && ...`. As you can see [here](https://github.com/HedvigInsurance/web-onboarding/pull/890/files#diff-e9fa83e568fe0a1210f47c062d4d43bb35e20fdcef8ac7f7d0ef6978fa685207R183)

Reply:
I forgot to pass `sticky` to the styled element, so `sticky` was `falsy`. Now that @guilhermespopolin forwards `sticky` the condition must be switched